specs/cm/change-mgt-vocab.ttl

```diff
@@ -28,10 +28,15 @@
 oslc_cm:
   a owl:Ontology ;
   dcterms:title "The OSLC Change Management(CM) Vocabulary" ;
+  rdfs:label "Change Management(CM)" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Change Management (CM) namespace."^^rdf:XMLLiteral ;
-  dcterms:source <https://tools.oasis-open.org/version-control/svn/oslc-ccm/trunk/specs/change-mgt/change-mgt-vocab.ttl> ;
   vann:preferredNamespacePrefix "oslc_cm" ;
-  rdfs:label "Change Management(CM)" .
+  dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
+  dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/cm/change-mgt-vocab.ttl> ;
+  dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/cm/change-mgt-spec.html> ;
+  dcterms:hasVersion "PS01" .
```

Comment:
same here, let's keep it WD until then. plus, we'd have to change the URIs in lockstep to point to the published file location.

Reply:
Fixed.